### ISSUE TITLE
Incredibly long loading times when building and deploying project from Docker

### ISSUE THREAD

**Issue by dain98** [harsh98trivedi/links#10]
You can see my version here: https://me.dain.cafe/

I've tried rebuilding the container and it didn't help. I also heard from a few people that the avatar image doesn't load for them. 

Loading Properly:
![image](https://user-images.githubusercontent.com/27909219/211491737-c8ee74cb-120b-4861-bb72-bb8d8cd9c4b8.png)

Breaking:
![image](https://user-images.githubusercontent.com/27909219/211491794-33dfb34a-a7c7-42f1-9bc2-fd76cf178935.png)

Do you have any idea what could be causing this issue? Since I'm just modifying a .yml file and running everything else the exact same way, I honestly have no idea what the problem is. 

Thank you for your time!

**Comment by dain98** [comment]
Update: I found out that the image not loading was an issue with imgur and wsrv.nl, as shown [here](https://github.com/weserv/images/issues/258). I hosted the images myself and now it loads fine. 

Site load times are still an issue though.

**Comment by negimeister** [comment]
The issue seems to be that the livereload script tries to load for almost 2 minutes before timing out and loading the rest of the page.

**Comment by negimeister** [comment]
I found the culprit. The `-l` flag in the Dockerfile causes Jekyll to enable live reload. This should only be used for development. 

This is fine for most local deployments where the connection simply fails and the page continues to load, but in other scenarios such as proxying through Cloudflare, the connection needs to time out first, causing the page not to load for almost 2 minutes.

I created a PR here #12
